corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -1150,6 +1123,32 @@ hqDefine("cloudcare/js/form_entry/entrycontrols_full", function () {
         return form.displayOptions || {};
     }
 
+    /**
+     * Utility to render question as select2
```

Comment:
I'd include the precondition that in order to call this, the calling object must define `additionalSelect2Options`

Reply:
Or pass the additional-option-generating function in as a parameter, since it isn't needed by any other code.